playground/src/app/simple-call/simple-call.component.ts

```diff
@@ -61,6 +64,21 @@ export class SimpleCallComponent implements OnInit, OnDestroy {
     );
 
     try {
+      if (
+        this.extras !==
+          `{
+        "key": "value"
+      }` &&
+        this.extras
+      ) {
```

Comment:
com base na sugestão acima

```suggestion
      if (this.extras && this.extras !== this.initialExtras) {
```